src/commands/info.ts

```diff
@@ -68,20 +67,50 @@ const info = async (
         summary += `\t- Running on PID ${process.pid}\n\nThis bot is not sharded and can see ${cache}.`;
     }
 
+    const application = (await (
+        await fetch(
+            `https://discord.com/api/v10/applications/${
+                parent.client.user!.id
+            }/rpc`
+        )
+    ).json()) as {
+        flags: number;
+    };
+    const flags = {
+        GuildMembers:
+            (application.flags & ApplicationFlags.GatewayGuildMembers) ==
+                ApplicationFlags.GatewayGuildMembers ||
+            (application.flags & ApplicationFlags.GatewayGuildMembersLimited) ==
+                ApplicationFlags.GatewayGuildMembersLimited,
+        GuildPresences:
+            (application.flags & ApplicationFlags.GatewayPresence) ==
+                ApplicationFlags.GatewayPresence ||
+            (application.flags & ApplicationFlags.GatewayPresenceLimited) ==
+                ApplicationFlags.GatewayPresenceLimited,
+        MessageContent:
+            (application.flags & ApplicationFlags.GatewayMessageContent) ==
+                ApplicationFlags.GatewayMessageContent ||
+            (application.flags &
+                ApplicationFlags.GatewayMessageContentLimited) ==
+                ApplicationFlags.GatewayMessageContentLimited
+    };
+
     embed
         .setDescription(summary)
         .addFields({
             name: 'Privileged Intents',
-            value: [
-                GatewayIntentBits.GuildPresences,
-                GatewayIntentBits.GuildMembers,
-                GatewayIntentBits.MessageContent
-            ]
+            value: Object.keys(flags)
                 .map(
-                    (u) =>
-                        `- ${intents.has(u) ? `✅` : `❌`} \`${
-                            GatewayIntentBits[u]
-                        }\``
+                    (k) =>
+                        `- ${
+                            flags[
+                                k as
+                                    | 'GuildMembers'
+                                    | 'GuildPresences' as 'MessageContent'
+                            ]
+                                ? '✅'
+                                : '❌'
+                        } \`${k}\``
```

Comment:
review based on [this](https://github.com/Dqrshan/discord-debug/pull/6#issuecomment-1646582859)